repo/packages/H/hue/0/config.json

```diff
@@ -0,0 +1,66 @@
+{
+  "type": "object",
+  "properties": {
+    "hdfs": {
+      "description": "Hadoop specific configuration properties",
+      "type": "object",
+      "properties": {
+        "scheduler-uri": {
+          "description": "HDFS package artifact tarball (must begin with 'hdfs')",
+          "type": "string",
+          "default": "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/artifacts/dcos/0.1.1/hdfs-mesos-0.1.1.tgz"
+        }
+      },
+      "required": [
+        "scheduler-uri"
+      ]
+    },
+    "hue": {
+      "description": "Hue specific configuration properties",
+      "type": "object",
+      "properties": {
+        "configuration": {
+          "description": "A custom universally-accessible URL of a hue.ini file.",
+          "type": "string",
+          "default": "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/hue/hue.ini"
+        },
+        "port": {
+          "description": "A custom static port for hue, if desired. This assists with external load balancers to discover hue.",
+          "type": "integer",
+          "minimum": 0,
+          "default": 0
+        }
+      },
+      "required": [
+        "configuration",
+        "port"
+      ]
+    },
+    "httpfs": {
+      "description": "Httpfs specific configuration properties",
+      "type": "object",
+      "properties": {
+        "configuration": {
+          "description": "A custom universally-accessible URL of a httpfs-site.xml file.",
+          "type": "string",
+          "default": "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/hue/httpfs-site.xml"
+        },
```

Comment:
This hue package bundles a httpfs server (its the only way to address an HA hdfs install). Currently, both are served via port 0, but if a user wants a specific port for either (to support some other tool, etc) they can.